pkgs/development/libraries/physics/yoda/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, fetchurl, python2Packages, makeWrapper }:
+{ stdenv, fetchurl, python2Packages, root, makeWrapper }:
 
 stdenv.mkDerivation rec {
   name = "yoda-${version}";
-  version = "1.6.5";
+  version = "1.6.6";
 
   src = fetchurl {
     url = "http://www.hepforge.org/archive/yoda/YODA-${version}.tar.bz2";
-    sha256 = "1i8lmj63cd3qnxl9k2cb1abap2pirhx7ffinm834wbpy9iszwxql";
+    sha256 = "088xx4q6b03bnj6xg5189m8wsznhal8aj3jk40sbj24idm4jl5yg";
   };
 
   pythonPath = []; # python wrapper support
 
-  buildInputs = with python2Packages; [ python numpy matplotlib makeWrapper ];
+  buildInputs = with python2Packages; [ python numpy matplotlib root makeWrapper ];
```

Comment:
ROOT adds 433 mb to closure and the only practical reason to enable it is to have root2yoda and yoda2root utilities. The reverse dependencies of YODA don't care about those. So yoda-with-root is for using interactively when user need those extra conversion utilities (most often you don't need those, actually).